packages/babel-preset-react-app/utils.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+const validateBoolOption = (name, value, defaultValue) => {
```

Comment:
Copy paste is your friend! Although I don't care

Reply:
As in you'd rather it be copypasta instead of a util file? 😄

Reply:
If it's something trivial like this, yeah I don't mind copy pasta.